plugins/pyembed/pyembed.cpp

```diff
@@ -1178,8 +1189,8 @@ class GILBlock
 class PythonRowStream : public CInterfaceOf<IRowStream>
 {
 public:
-    PythonRowStream(PythonThreadContext *_sharedCtx, PyObject *result, IEngineRowAllocator *_resultAllocator)
-    : sharedCtx(_sharedCtx), resultIterator(NULL)
+    PythonRowStream(PyObject *result, IEngineRowAllocator *_resultAllocator)
+    : resultIterator(NULL)
```

Comment:
double checking, so when getDatasetResult is called which constructs a PythonRowStream, GIL lock is already done somewhere?